apps/web/src/ee/clerk/providers/useCreateAuthEnterprise.ts

```diff
@@ -38,7 +38,11 @@ export function useCreateAuthEnterprise() {
   const { signOut, orgId } = useAuth();
   const { user: clerkUser, isLoaded: isUserLoaded } = useUser();
   const { organization: clerkOrganization, isLoaded: isOrganizationLoaded } = useOrganization();
-  const { setActive, isLoaded: isOrgListLoaded } = useOrganizationList();
+  const {
+    setActive,
+    isLoaded: isOrgListLoaded,
+    userMemberships,
+  } = useOrganizationList({ userMemberships: { infinite: true } });
```

Comment:
`infinite: true` means that when `fetchNext()` is called, it gets appended to existing `userMemberships.data` property

https://clerk.com/docs/references/react/use-organization-list